lib/add-project.js

```diff
@@ -0,0 +1,25 @@
+const { collectAndSaveProject } = require('./init/project');
+const projectsToText = require('./utils/projectsToText');
+const getSelectedProjects = require('./utils/getSelectedProjects');
+
+function quit() {
+  console.log('Project selection was not updated.');
+  process.exitCode = 2;
+  process.exit();
+}
+
+const addProject = async () => {
+  const projects = getSelectedProjects();
+
+  try {
+    console.log(`\nYou're currently set up to sync text from the following projects: ${projectsToText(projects)}`);
+    await collectAndSaveProject(false);
+  } catch (error) {
+    if (error && error.response && error.response.status === 400) {
```

Comment:
Good catch! I think we definitely do.

It seems like it'd be better for us to err (ha!) on the side of always showing the error to the user instead of swallowing information based on the status code.

In looking at the implementation of `collectAndSaveProject()`, it does like we usually catch errors and `quit()` in places where they might be expected, so that further reinforces my thought that we should just show the error message here. I'm going to change the implementation to work that way, but let me know if you have any thoughts about why we might want to abstract the error itself and `console.log` a message instead (like the current code)!

Reply:
It also just occurred to me that the error message itself is inaccurate, since this `catch` is catching errors thrown from the process of adding a project 😅 I'm going to update that too!